### DISCUSSION TITLE
O-RAN Specifications

### DISCUSSION THREAD

**Discussion by alirezalab134** [srsran/srsRAN_Project#870] [General Help] [answered]
Hi Everyone,
When I was trying to change the ran controller command in the example xApp of the oran-sc-ric (simple_rc_xapp.py), I realized the ASN1 command is not matching the latest version of the O-RAN.WG3.E2SM-RC-R003 on O-RAN specification website. Based on the [tutorial](https://docs.srsran.com/projects/project/en/latest/tutorials/source/near-rt-ric/source/index.html#o-ran-nearrt-ric-and-xapp) they wrote the E2 term based on the version 3 of the mentioned document.
However I was not able to find that specific version online. Can anyone help me find those documents?
- O-RAN.WG3.E2AP-R003-v03.00
- O-RAN.WG3.E2SM-R003-v03.00
- O-RAN.WG3.E2SM-KPM-R003-v03.00
- O-RAN.WG3.E2SM-RC-R003-v03.00

**Comment by hudouseko** [comment, marked as answer]
Hi @alirezalab134 , go to the  [ORAN specification download page](https://specifications.o-ran.org/), Select workgroup 3, turn off "Show latest" and "Apply filter". That will show you as well older spec versions.

**Reply by alirezalab134** [reply]
Thanks, that worked.
